### ISSUE TITLE
Request: create a WAN game visible on server list

### ISSUE THREAD

**Issue by Pretuer** [Unvanquished/Unvanquished#1263]
I think that WAN games should be visible on server list for matchmaking and/or tournments

**Comment by necessarily-equal** [comment]
For an option to turn LAN games into WAN games, visible from the server list, we arguably need two thinks:

1. An option to turn on advertising the game on the master server (that will start daemond with `/set server.private 0`)
2. Either an explaination that you need to open a port, or some UPnP magic

**Comment by Gireen** [comment]
2. [STUN](https://en.wikipedia.org/wiki/STUN) seems to be another method to solve this problem.

**Comment by illwieckz** [comment]
One side thing we may want to make the experience good even if we would already have a button to enable server advertising and some _abracadabra_ to open ports, is a map ballot screen at the end of the game, so not having a configured rotation map would work. Currently the server just restarts the current map. Note: we may also include a default map rotation but that is not an argument against a map ballot screen.

**Comment by Gireen** [comment]
@illwieckz not the scope ot this issue 🤔 #1016 

And i think STUN is not so much magic. If i understand it correct the game sends a udp package to a server on the desired port and trough that the NAT lets this port open to the outside.

**Comment by illwieckz** [comment]
> not the scope ot this issue

Yes, I'm just making a remark about the fact that any technical solution to open port, get public ip, advertise server etc. will not make user happy until we polish the server hosting experience.

**Comment by slipher** [comment]
https://github.com/DaemonEngine/Daemon/pull/1395 removes the ROM flag preventing you from configuring the server to be public in the client.

Other than that, I don't think there's anything more we can do since we can't automate asking the ISP to enable port forwarding or whatever.

**Comment by Gireen** [comment]
So are STUN or UDP hole punching not possible? 

Its probably one of the lowest priorities but interesting as a feature for less technical players.
